packages-next/types/src/index.ts

```diff
@@ -142,12 +136,41 @@ export type KeystoneSystem = {
   views: string[];
 };
 
+export type AccessControlContext = {
+  getListAccessControlForUser: any; // TODO
+  getFieldAccessControlForUser: any; // TODO
+};
+
 export type SessionContext = {
-  session: any;
+  // Note: session is typed like this to acknowledge the default session shape
+  // if you're using keystone's built-in session implementation, but we don't
+  // actually know what it will look like.
+  session?:
+    | {
+        itemId: string;
+        listKey: string;
+        data?: Record<string, any>;
+      }
+    | any;
```

Comment:
This `| any` means the the type is just `any`. If you want to leave the previous thing there for documentation, that's fine but just be aware that it's doing nothing type-wise

Reply:
Yeah, I know, and I don't know what to do about that 😞 

Would prefer to leave it there for now so if someone is peeking the types, they find it, but we'll probably remove it later when we put another round of focus into generating this stuff.